packages/core/test/lib/utils/spanUtils.test.ts

```diff
@@ -85,6 +85,9 @@ describe('spanToJSON', () => {
       trace_id: 'abcd',
       origin: 'auto',
       start_timestamp: 123,
+      data: {
+        'sentry.op': 'test op',
+      },
```

Comment:
I noticed that `spanToJson` sets the op also in `data` because it's saved as an attribute. Should we change this in the `spanToJson` implementation? I'd imagine, this would just be duplicated data in the product as we still need the top level `event.op` field anyway. 
thoughts @mydea @AbhiPrasad?

Reply:
I would leave it in, I think - rather if we don't want to send this, we can/should remove it wherever we generate the event 🤔 eventually (after v8? let's see..) I think any special handling of `op` & `origin` should go away and they should _only_ be kept as attributes everywhere. But for now we can just keep both I believe...?

Reply:
> eventually (after v8? let's see..) I think any special handling of op & origin should go away and they should only be kept as attributes everywhere

Agreed, that makes sense to me. Ok, I'll leave it in then and adjust the tests to accomodate the new behaviour. Worth pointing out that this isn't breaking anything, it's just adding a new data item anyhow.